uplink/src/base/mod.rs

```diff
@@ -94,7 +92,10 @@ pub struct Config {
     pub ota: Ota,
     pub stats: Stats,
     pub simulator: Option<SimulatorConfig>,
-    pub journalctl: Option<JournalctlConfig>
+    #[cfg(target_os = "linux")]
+    pub journalctl: Option<JournalctlConfig>,
+    #[cfg(target_os = "android")]
+    pub run_logcat: bool,
```

Comment:
Maybe we should have `LogcatConfig` instead of `run_logcat`. Or We could have a `LoggerConfig` here and use that for both.